src/Modal/Modal.component.tsx

```diff
@@ -49,7 +49,9 @@ const StyledModal = StyledReactModal.default.styled`
     // Footer (respectively) are also used.
     ${StyledContent} {
         padding: ${({ size = defaultSize }: ModalProps) =>
-            `4rem ${Sizes[size].contentPadding}rem 0 ${Sizes[size].contentPadding}rem`};
+            `4rem ${Sizes[size].contentPadding}rem 0 ${
```

Comment:
Just curious: what version of prettier are you on locally?

Reply:
`prettier@1.17.1`